pugjs/engine.go

```diff
@@ -90,6 +92,7 @@ func NewEngine() *Engine {
 	return &Engine{
 		RWMutex:      new(sync.RWMutex),
 		TemplateCode: make(map[string]string),
+		ratelimit:    make(chan struct{}, 8),
```

Comment:
Default?

Reply:
i personally think that the default value should be the old hard-configured value to prevent behavior changes in existing projects.

in general it is good have it configurable to arrange the value to the project-template-complexety and the available cpu-resources and to make it transparent at all.

after rethinking about my thought that it can effect the total amount of cpu usage, i need to revoke this assumption, because we time-independent parallelism strategy.